compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -822,6 +746,22 @@ class Namer { typer: Typer =>
       else completeInCreationContext(denot)
     }
 
+    protected def addAnnotations(sym: Symbol): Unit = original match {
+      case original: untpd.MemberDef =>
+        var hasInlineAnnot = false
+        lazy val annotCtx =
```

Comment:
While you're at it, you may want to make an annotCtx def that can be used both here and for https://github.com/lampepfl/dotty/blob/64bd0fe5210cacd7c17ab285c80712944157c212/compiler/src/dotty/tools/dotc/typer/Typer.scala#L1345 :)